matrix-room.c

```diff
@@ -1020,10 +1020,8 @@ void matrix_room_send_message(PurpleConversation *conv, const gchar *message)
         return;
```

Comment:
> In fact it looks like we shouldn't be doing the unescaping on this path?

We shouldn't, and we aren't. The image itself requires no unescaping. The unescaped form of the message is only needed for the `m.text` message body, so moving `message_to_send` assignment after the image-handling will address this.